docs/supported_apis/dataframe_supported.rst

```diff
@@ -103,15 +111,16 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``dot``                    | `dot`_                    | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``drop``                   | `drop`_                   | Y                      |                                                    |
+| ``drop``                   | `drop`_                   | Y                      | **Omnisci**:``P``, ``D`` for row drop              |
```

Comment:
```suggestion
| ``drop``                   | `drop`_                   | Y                      | **Omnisci**: ``P``. ``D`` for row drop              |
```
There might be confusion whether P and D are for row drop or only D is for row drop. This should probably be rephrased or left as is with the suggestion because I think it looks clearer.

Reply:
Fixed hopefully